src/app/_pages/join/join.page.scss

```diff
@@ -0,0 +1,60 @@
+mat-grid-tile {
+    background: #0277bd;
+    border-radius: 5px;
+}
+
+button { 
+    position: absolute; 
+    bottom: 5%;
+    right: 5%;
+    border: 0px solid #d6d6d6;
+    width: 150px;
+    height: 50px;
+}
+
+mat-selection-list {
+    display: grid;
```

Comment:
ok, so you went with display grid, instead of flexbox with wrapping. I think in order to get things to not overflow the page width you're going to have to set media query breakpoints and change your grid arrangement. This is more effort than I would put in (I'd use flexbox, allow it to wrap, set an element width, and set an element max width of 100%, and let the elements fill the space/wrap), but you can do it this way if you want to have that level of control over it. I'm not very experienced at grid so there may be a trick I'm missing with it.